### ISSUE TITLE
Using low-ell likelihoods with trained model

### ISSUE THREAD

**Issue by AndreasNygaard** [AarhusCosmology/connect_public#14]
There is an issue when using `Planck_lowl_TT` and `Planck_lowl_EE` with a neural network trained on an iteratively sampled data set, but for different reasons. 

The `Planck_lowl_TT` likelihood fails when using regular CLASS (and not CLASS++) because `cosmo.T_CMB()` returns 0 in the MontePython plugin since `cosmo.compute()` has not been called. A way to solve this would be to simply call `super(Class).compute(level=['background'])` when `cosmo.T_cmb()` is called.

The `Planck_lowl_EE` likelihood fails (it crashes the run and does not just output a very negative likelihood) when using a network trained on an iteratively sampled data set, since only the `Planck_highl_TTTEEE_lite` likelihood has been used during training. This favours too large values of `tau_reio`, and we thus get too high values of the EE spectrum at low multipoles. This crashes the likelihood (a bug in the `clik` code), and the mcmc run thus crashes too. A way to fix this, would be to always use the low-ell likelihoods for the iterative sampling of training data. Furthermore, when including low-ell likelihoods as well as the "cosmic_variance" loss function, one may have to retrain the final neural network with more epochs, as to not have too low an accuracy in the low multipoles.
